src/bindings/python/flux/uri/resolvers/lsf.py

```diff
@@ -0,0 +1,84 @@
+###############################################################
+# Copyright 2022 Lawrence Livermore National Security, LLC
+# (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+#
+# This file is part of the Flux resource manager framework.
+# For details, see https://github.com/flux-framework.
+#
+# SPDX-License-Identifier: LGPL-3.0
+###############################################################
+
+"""
+This module supports the Flux URI resolver for IBM Spectrum LSF.
+
+It can be used like:
+  `flux uri --remote lsf:<JOBID>`.
+
+"""
+
+import os
+import re
+import subprocess
+from pathlib import PurePath
+
+import yaml
+
+from flux.uri import URIResolverPlugin
+
+
+def lsf_find_compute_node(jobid):
+    """Figure out where the job is being run using JSON output from `bjobs -l`."""
+
+    sp = subprocess.run(
```

Comment:
I think it would be nice to add `check=True` to any of your `subprocess.run` calls, because that will throw an error with the captured stdout and stderr if the process exits with nonzero return code.

For instance, it makes sense to me that before trying to parse the YAML output of the CSM command, you would check whether the CSM command exited normally. That way you can spit out whatever error message CSM gave you, rather than throwing a YAML decode error.